webpack.config.js

```diff
@@ -6,7 +6,7 @@ var gulpConfig = require('./gulp/config');
 var CopyWebpackPlugin = require('copy-webpack-plugin');
 
 module.exports = {
-	devtool: 'source-map',
+	devtool: 'eval',
```

Comment:
Snuck in this, makes webpack watching way faster but might make source maps a little worse.